@navikt/ds-react/layouts/product-page/Panel.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Hva tenker du om å fristille Panel-komponenten fra ProductPageLayout så den kan brukes andre steder også?

Reply:
tja, godt spørsmål. Jeg trudde det allerede eksisterte et panel i designsystemet? :) 
Den inkluderer jo funksjonaliteten "Kopier lenke" som antageligvis er ganske skreddersydd for Layout, og mulig vi ønsker å gjøre andre tilpasninger som kun gjelder layout i fremtiden? Jeg er usikker, er gode argumenter for å dra den ut også :)

Reply:
Ken og jeg tar en vurdering her ila. uka.